docs/pages/reference.mdx

```diff
@@ -105,3 +105,14 @@ if (import.meta.env.DEV) {
 ```
 
 In the minimal application you get from `pnpm create mud@<version> <app>`, the MUD dev tools are already included.
+
+## MUD server components
```

Comment:
it seems like this should be part of the `## Debugging` category, so I would expect the headline to be `### Server components` (akin to `### Onchain components`)
```suggestion
### Server components
```